lib/IRGen/GenFunc.cpp

```diff
@@ -1286,6 +1286,7 @@ class AsyncPartialApplicationForwarderEmission
     return subIGF.Builder.CreateCall(fnPtr.getAsFunction(subIGF),
                                      asyncExplosion.claimAll());
   }
+  // [FIXME: swiftasynccc] This call should be marked musttail.
```

Comment:
[In case anyone else ends up looking at this] The reason for this FIXME is that Arnold suggested we should handle this separately as it might cause trouble; as we have some known issues with certain kinds of forwarders.